OBJECT-SERVER.md

```diff
@@ -0,0 +1,27 @@
+# Using Realm Object Server on Android
+
+
+1) Add the following to the top-level `build.gradle` file:
+
+
+```groovy
+buildscript {
+repositories {
+jcenter()
+}
+dependencies {
+classpath "io.realm:realm-gradle-plugin:2.0.0"
```

Comment:
This is going to be a pain to keep updated so I'd prefer to just have a placeholder instead. Thoughts?